package.json

```diff
@@ -17,6 +17,7 @@
     "chromedriver": "^86.0.0",
     "cssnano": "^4.1.10",
     "eslint": "^7.13.0",
+    "eslint-config-prettier": "^6.15.0",
```

Comment:
https://github.com/prettier/eslint-config-prettier - turns off rules that would conflict between eslint and prettier